screens/Login.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+import { TouchableOpacity, TextInput, Text, View, StyleSheet } from 'react-native'
```

Comment:
When you import more than 2 exports from one file you should organize your imports in a column.